esp-hal/src/gpio/lp_io.rs

```diff
@@ -191,77 +195,76 @@ macro_rules! lp_gpio {
     (
         $($gpionum:literal)+
     ) => {
-        paste::paste!{
-            $(
-                impl $crate::gpio::RtcPin for GpioPin<$gpionum> {
-                    unsafe fn apply_wakeup(&mut self, wakeup: bool, level: u8) {
-                        let lp_io = &*$crate::peripherals::LP_IO::ptr();
-                        lp_io.[< pin $gpionum >]().modify(|_, w| {
-                            w.wakeup_enable().bit(wakeup).int_type().bits(level)
+        $(
+            impl $crate::gpio::RtcPin for GpioPin<$gpionum> {
```

Comment:
With the new PAC update, this impl can be moved out of the macro. I'm hoping to see this in some future PR.